packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailViewProvider.tsx

```diff
@@ -72,7 +72,8 @@ export function LocationDetailViewProvider({
     message: hasError ? message : downloadErrorMessage,
   });
 
-  const isNoActionAvailable = actions.every((obj) => obj.isHidden);
+  const isNoActionAvailable =
+    !actions || actions?.every((action) => action.isHidden);
```

Comment:
Probably don't need the optional changing since we've already checked for truthiness of `actions`. On that note think this could be condensed to:
```
!actions?.every((action) => action.isHidden);
```